MongoDB.FSharp.Tests/AcceptanceTests.fs

```diff
@@ -41,173 +41,205 @@ type DimmerSwitch =
     | On
 
 type ObjectWithOptions() =
-    member val Id : BsonObjectId = BsonObjectId.GenerateNewId() with get, set
+    member val Id : BsonObjectId = BsonObjectId(ObjectId.GenerateNewId()) with get, set
     member val Age : int option = None with get, set
 
 type ObjectWithDimmer() =
-    member val Id : BsonObjectId = BsonObjectId.GenerateNewId() with get, set
+    member val Id : BsonObjectId = BsonObjectId(ObjectId.GenerateNewId()) with get, set
     member val Switch : DimmerSwitch = Off with get, set
 
 type ObjectWithDimmers() =
-    member val Id : BsonObjectId = BsonObjectId.GenerateNewId() with get, set
+    member val Id : BsonObjectId = BsonObjectId(ObjectId.GenerateNewId()) with get, set
     member val Kitchen : DimmerSwitch = Off with get, set
     member val Bedroom1 : DimmerSwitch = Off with get, set
     member val Bedroom2 : DimmerSwitch = Off with get, set
 
 type ``When serializing lists``() = 
-    let db = MongoDatabase.Create "mongodb://localhost/test"
+    let conn = new MongoClient("mongodb://localhost")
+    let db = conn.GetDatabase("test")
     do
         Serializers.Register()
 
     interface System.IDisposable with
         member this.Dispose() = 
-            db.DropCollection "objects" |> ignore
-            db.DropCollection "persons" |> ignore
+            ()
+            db.DropCollectionAsync "objects" |> AwaitVoidTask |> ignore
+            db.DropCollectionAsync "persons" |> AwaitVoidTask |> ignore
 
     /// Seems to be fixed in version 1.5 of the C# driver
     [<Fact>]
     member this.``It can serialize an object with a list``() =
-        let collection = db.GetCollection<ObjectWithList> "objects"
-        let obj = ObjectWithList()
-        obj.List <- [ "hello"; "world" ]
-        collection.Save obj |> ignore
-
-        let genCollection = db.GetCollection "objects"
-        let fromDb = genCollection.FindOne(new QueryDocument("_id", obj.Id))
-        let array = fromDb.["List"].AsBsonArray
-        Assert.Equal(2, array.Count)
+            async {
```

Comment:
Yes in fact i just removed them as your suggestion, i added those during my debugging with mongo source code, debugger wan't stepping into and i added those because i tought i was doing something wrong with Async.Await..